primitives/identifier/src/curi.rs

```diff
@@ -190,7 +190,7 @@ impl Ss58Identifier {
 		Ok(Self(
 			Vec::<u8>::from(v.to_base58())
 				.try_into()
-				.map_err(|_| IdentifierError::InvalidIdentifierLength)?,
+				.map_err(|_| IdentifierError::InvalidIdentifier)?,
```

Comment:
This is being triggered even when vector length is set to 40.
Ideally should be returned OK()

Reply:
@vatsa287 open a separate issue for this, and lets move ahead with merging this testcase, which checks for `IdentifierLength`.